<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NearMissWindow</class>
 <widget class="QMainWindow" name="NearMissWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Near Misses</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>571</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="spectVerticalLayout"/>
   </widget>
   <widget class="QPushButton" name="previousBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Vorherige</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>NÃ¤chste</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>pausieren</string>
    </property>
   </widget>
   <widget class="QListWidget" name="audioFileList">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
border: 1px solid black;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playAudioBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>abspielen</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
